runners/apex/src/main/java/org/apache/beam/runners/apex/ApexRunner.java

```diff
@@ -72,6 +78,12 @@
    */
   public static final AtomicReference<AssertionError> ASSERTION_ERROR = new AtomicReference<>();
 
+  /**
+   * The runner will look for this file in the classpath for Apex configuration properties
+   * unless an alternative location was specified in pipeline options.
+   */
+  public static final String RUNNER_PROPERTIES_RESOURCE = "/beam-runners-apex.properties";
```

Comment:
Yes, wanted to do that also, would be better from a documentation perspective. But a value starting with '/' is interpreted as local file path, while the default is a classpath resource.  Any idea how to handle that?